baseapp/abci.go

```diff
@@ -177,7 +177,7 @@ func (app *BaseApp) Query(_ context.Context, req *abci.RequestQuery) (resp *abci
 
 	telemetry.IncrCounter(1, "query", "count")
 	telemetry.IncrCounter(1, "query", req.Path)
-	defer telemetry.MeasureSince(time.Now(), req.Path)
+	defer telemetry.MeasureSince(sdk.Context{}, time.Now(), req.Path)
```

Comment:
If this approach is used, it is necessary to consider how to handle this case.

Reply:
Thanks for the comment! This PR is part of [this comment](https://github.com/cosmos/cosmos-sdk/issues/10245#issuecomment-2020642779) I made on this issue: https://github.com/cosmos/cosmos-sdk/issues/10245